### ISSUE TITLE
Sub-Hire Tracking

### ISSUE THREAD

**Issue by tomtom5152** [nottinghamtec/PyRIGS#164]
Originally reported by: **Sam Osborne (BitBucket: [samozzy](http://bitbucket.org/samozzy), GitHub: [samozzy](http://github.com/samozzy))**

---

Add, like Dry-Hires, a field to track sub-hire. Just to make sure everyone knows when it's coming and is due to go back. Very useful for when we have lots of subhire eg W1, Grad Ball and saves on spreadsheets.

---
- Bitbucket: https://bitbucket.org/nottinghamtec/pyrigs/issue/165

**Comment by samozzy** [comment]
Endsleigh wants the following data:
•         Details of Equipment
•         Sum Insured
•         Period of cover
•         Referred to Endsleigh (Yes or No) if over the blanket endorsement limit _I do not know what this is yet_
This is trivial to make into a Google Form or some such, and then link to from RIGS, but Google Forms doesn't offer file uploads (of the quote), whereas RIGS could.

**Comment by toastmaster2k2** [comment]
You can include links to documents on Google drive on a google doc/sheet.  Rigs would be neater in the user end I would assume.

[cid:cm_1461067265785_1]http://www.nottinghamtec.co.uk/               Joseph Heil
TEC PA & Lightinghttp://www.nottinghamtec.co.uk/
Mobile: 07970430271
Office Tel: 0115 84 68720tel:01158468720
Emergency Mobile: 07825 065681
Portland Building, University Park, Nottingham, NG7 2RD

via CloudMagic Emailhttps://cloudmagic.com/k/d/mailapp?ct=pa&cv=8.6.36&pv=6.0&source=email_footer_2
On Mon, Aug 08, 2016 at 12:27 PM, Sam Osborne <notifications@github.com<mailto:notifications@github.com>> wrote:

Endsleigh wants the following data:
- Details of Equipment
- Sum Insured
- Period of cover
- Referred to Endsleigh (Yes or No) if over the blanket endorsement limit I do not know what this is yet
  This is trivial to make into a Google Form or some such, and then link to from RIGS, but Google Forms doesn't offer file uploads (of the quote), whereas RIGS could.

##

**Comment by samozzy** [comment]
Made a thing - https://form.jotformeu.com/62203600438344
Google Forms don't allow for file submission, but this integrates with Google Drive so we get a spreadsheet and all file uploads going there. A TEC account should be made with jotform, as it doesn't have useful (read: free) form sharing, but the Google Drive folder with uploads can be shared as normal.

**Comment by harry-bridge** [comment]
@samozzy I still feel we shouldn't be made to do this, as there is literally a man in the SU whose job this is, and as we don't deal with endsleigh directly we can't be sure whether anything actually gets insured

**Comment by tomtom5152** [comment]
@pickleshb Further to that, I know Ryan was in fact serverly bollocked for dealing with Endsleigh directly

**Comment by samozzy** [comment]
I agree - we shouldn't deal with Endsleigh directly. From my understanding, the blanket insurance policy covers us up to X (which I need to find out). For everything we hire under X, we don't _need_ to tell Stephen about, but need to make a note of as at any point Endsleigh may ask the SU for it to change the premium. Obviously, there's no harm in telling him, but someone needs to keep the record (ideally both of us, so that what has just happened last week doesn't again).
Anything over X we must tell Stephen about well in advance of the hire period to ensure it is insured.

**Comment by FreneticScribbler** [comment]
This has come back up recently. Some way to make the subhire form more obvious, in particular.
